superduperdb/db/mongodb/cdc/vector_task_factory.py

```diff
@@ -1,7 +1,7 @@
 import typing as t
 
 from superduperdb.container.serializable import Serializable
-from superduperdb.vector_search.base import VectorCollectionConfig, VectorCollectionItem
+from superduperdb.vector_search.base import VectorSearchConfig, VectorItem
 
```

Comment:
lets remove this file, already in update_tasks I reckon.

Reply:
Right, point taken.

Reply:
Done